be/src/olap/memtable_memory_limiter.cpp

```diff
@@ -225,10 +225,8 @@ void MemTableMemoryLimiter::refresh_mem_tracker() {
     _log_timer.reset();
     // if not exist load task, this log should not be printed.
     if (_mem_usage != 0) {
-        LOG(INFO) << ss.str() << ", process mem: " << PerfCounters::get_vm_rss_str()
-                  << " (without allocator cache: "
-                  << PrettyPrinter::print_bytes(GlobalMemoryArbitrator::process_memory_usage())
-                  << "), load mem: " << PrettyPrinter::print_bytes(_mem_tracker->consumption())
+        LOG(INFO) << ss.str() << ", " << GlobalMemoryArbitrator::process_memory_used_details_str()
```

Comment:
better do not use streamstring，use fmt to same work. the stringstream bad performance